tests/e2e/performance/test_pvc_creation_performance.py

```diff
@@ -11,13 +11,35 @@
 from ocs_ci.framework.testlib import performance, E2ETest, polarion_id, bugzilla
 from ocs_ci.helpers import helpers
 from ocs_ci.ocs import defaults, constants
-
+from ocs_ci.utility.utils import exec_cmd
+from ocs_ci.ocs.resources.pvc import (
+    get_all_pvcs_in_storageclass,
+    get_all_pvs_in_storageclass,
+)
 
 from ocs_ci.utility.performance_dashboard import push_to_pvc_time_dashboard
+from ocs_ci.ocs.cluster import CephCluster
 
 log = logging.getLogger(__name__)
 
 
+def get_storage_pool_name(interface, namespace):
+    """
+    Return the pool name of a storageclass interface
+
+    Args:
+        interface (str): the name of the storageclass interface
+        namespace (str): the name of the namespace where the storageclass define
+
+    Returns:
+        str : the name of the pool
+
+    """
+    cmd = f"oc get {interface} -n {namespace} -o jsonpath='{{.items[0].metadata.name}}'"
```

Comment:
you may need to handle cases where there is more than one pool for an interface.